src/HealthcareApis/HealthcareApis/help/Remove-AzHealthcareApisService.md

```diff
@@ -0,0 +1,201 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.HealthcareApis.dll-Help.xml
+Module Name: Az.HealthcareApis
+online version: https://docs.microsoft.com/en-us/powershell/module/az.healthcareapisservice/remove-azhealthcareapisservice
+schema: 2.0.0
+---
+
+# Remove-AzHealthcareApisService
+
+## SYNOPSIS
+Deletes a service instance.
+
+## SYNTAX
+
+### ServiceNameParameterSet (Default)
+```
+Remove-AzHealthcareApisService -Name <String> -ResourceGroupName <String> [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### InputObjectParameterSet
+```
+Remove-AzHealthcareApisService [-InputObject <PSHealthcareApisService>] [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+### ResourceIdParameterSet
+```
+Remove-AzHealthcareApisService [-ResourceId] <String> [-AsJob] [-DefaultProfile <IAzureContextContainer>]
+ [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Deletes a service instance.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Remove-AzHealthcareApisFhirService -Name MyService -ResourceGroupName MyResourceGroup
+
+ResourceGroupName Name 
+----------------- ----------- 
+MyResourceGroup   MyService
+```
+
+Deletes the existing HealthcareApis service with the provided name within a provided resource group.
+
+### Example 2
+```powershell
+PS C:\> Remove-AzHealthcareApisFhirService -ResourceId myResourceId
```

Comment:
@poadhika this example is incorrect -- `myResourceId` would be interpreted as a string that would then be parsed as a resource ID string, which would fail. You need to have a sample resource ID assigned to a variable that you then pass to this parameter:

```powershell
$ResourceId = "/subscriptions/xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx/resourceGroups/MyResourceGroup/providers/Microsoft.HealthcareApis/services/MyService
Remove-AzHealthcareApisService -ResourceId $ResourceId
```